backend/src/middleware/klicktippMiddleware.ts

```diff
@@ -12,7 +12,7 @@ import CONFIG from '../config/index'
 //   // Do Something here before resolver is called
 //   const result = await next()
 //   // Do Something here after resolver is completed
-//   await signIn(result.email, result.language, result.firstName, result.lastName)
+//   await klicktippSignIn(result.email, result.language, result.firstName, result.lastName)
```

Comment:
Doesn't we need this anymore?
When did it get commented out?

Reply:
I do not know. This is your code.